extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-postgresql/src/main/java/org/apache/guacamole/auth/postgresql/PostgreSQLGuacamoleProperties.java

```diff
@@ -92,6 +92,30 @@ private PostgreSQLGuacamoleProperties() {}
 
     };
 
+    /**
+     * Sets the number of seconds the driver will wait for
```

Comment:
This is a bit of a nitpick, but the below statement doesn't actually "set" anything.  It is a property that can be used to set it.  Take a look at some of the other properties around this - for example, "The password used to authenticate..."  This can probably be simplified to something similar to that.